### PR TITLE
feat(log): configure log paths through env by default

### LINKED ISSUE #694: Make server log path configurable via .env

### PR DESCRIPTION
**Pull request by kevinfrom**
Closes #694 

- What do you think of the naming of the env variables?
- Will calling `$this->kernel->loadConfig()` cause any issues for other tests?

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -33,6 +33,8 @@
     <env name="BASE_URI" value="" />
     <env name="CACHE" value="null" />
     <env name="DISCOVERY_CACHE" value="true" />
+	<env name="LOG_PATH_DEBUG" value="log/debug.test.log" />
+	<env name="LOG_PATH_SERVER" value="log/server.test.log" />
```
Comment (innocenzi) [inline phpunit.xml.dist:37]:
```suggestion
    <env name="LOG_PATH_DEBUG" value="log/debug.test.log" />
    <env name="LOG_PATH_SERVER" value="log/server.test.log" />
```

File: src/Tempest/Log/src/Config/logs.config.php [hunk 2]
```diff
@@ -5,5 +5,9 @@
 namespace Tempest\Config;
 
 use Tempest\Log\LogConfig;
+use function Tempest\env;
 
-return new LogConfig();
+$debugLogPath = env('LOG_PATH_DEBUG');
+$serverLogPath = env('LOG_PATH_SERVER');
+
+return new LogConfig(debugLogPath: $debugLogPath, serverLogPath: $serverLogPath);
```
Comment (innocenzi) [inline src/Tempest/Log/src/Config/logs.config.php:13]:
```suggestion
return new LogConfig(
    debugLogPath: env('LOG_PATH_DEBUG'),
    serverLogPath: env('LOG_PATH_SERVER'),
);
```

**Review by innocenzi — DISMISSED** [review]
Minor style nitpicks

File: .env.example [hunk 3]
```diff
@@ -18,4 +18,8 @@ CONFIG_CACHE=false
 VIEW_CACHE=false
 
 # Enable or disable project cache (allround cache)
-PROJECT_CACHE=false
\ No newline at end of file
+PROJECT_CACHE=false
+
+# Overwrite default log paths (null = default)
+LOG_PATH_SERVER=null
```
Comment (brendt) [inline .env.example:24]: I'd call them `SERVER_LOG_PATH` and `DEBUG_LOG_PATH`

**Comment by brendt** [comment]
> $this->kernel->loadConfig()

I think it should be fine :)

**Comment by brendt** [comment]
Please take a look at the review comments, apart from that this is good to merge :)

**Comment by kevinfrom** [comment]
All request changes have been resolved :) Thanks for the feedback.

**Comment by brendt** [comment]
For future reference: please run `composer qa` locally to submit codestyle changes. No worries about it for now, I ran it for you 👍

**Pull request merged**